packages/abseil_cpp.rb

```diff
@@ -3,11 +3,10 @@
 class Abseil_cpp < Package
   description 'Abseil Common Libraries C++'
   homepage 'https://abseil.io/'
-  @_ver = '20200923.3'
-  version @_ver
+  version '20200923.3'
   license 'Apache-2.0'
   compatibility 'all'
-  source_url "https://github.com/abseil/abseil-cpp/archive/#{@_ver}.tar.gz"
+  source_url "https://github.com/abseil/abseil-cpp/archive/#{@version}.tar.gz"
```

Comment:
Wouldn't `#{version}` work here?

Reply:
Both `version` and `@version` should be identical

Reply:
(Removing `@` as it is simpler...)